sass/includes/_filters.scss

```diff
@@ -0,0 +1,127 @@
+.filters {
+    $root: &;
+    display: flex;
+    gap: 2rem;
+    padding: 1.5rem 0;
+    flex-direction: column;
+
+    @media only screen and (min-width: #{$screen__md + 1px}) {
+        align-items: flex-start;
+    }
+
+    @media only screen and (min-width: #{$screen__lg + 1px}) {
+        flex-direction: row;
+        gap: 1rem;
+    }
+
+    @media only screen and (min-width: #{$screen__xl + 1px}) {
+        gap: 2rem;
+    }
+
+    &__fieldset {
+        flex-shrink: 0;
+        border: 1px solid $color__black;
+    }
+
+    &__super-title {
+        font-family: $font__open-sans;
+        font-weight: 700;
+        font-size: 1rem;
+        background-color: transparent;
+        padding: 1rem 1rem 0.5rem;
+        margin: 0;
+        box-sizing: border-box;
+        float: left;
+        width: 100%;
+    }
+
+    &__label {
+        font-family: $font__open-sans;
+        font-weight: 700;
+        font-size: 1rem;
+        margin-bottom: 0;
+
+        &--top {
+            display: block;
+        }
+    }
+
+    &__help {
+        margin: 0 0 0.5rem;
+        color: $color__grey-800;
+        font-size: 0.875rem;
+    }
+
+    &__radio-list {
+        display: flex;
+        flex-direction: column;
+        gap: 0.5rem;
+    }
+
+    &__panel {
+        padding: 1rem;
+        box-sizing: border-box;
+    }
+
+    &__date {
+        min-height: 1.5rem;
+        width: auto;
+        accent-color: $color__black; // otherwise the text is yellow in iOS
+        color: $color__black;
+    }
+
+    &__toggle {
+        flex-shrink: 0;
+    }
+
+    &__toggle-label {
+        display: flex;
+        align-items: center;
+        color: $color--grey-700;
+        font-size: 1rem;
+        justify-content: space-between;
+
+        @media only screen and (min-width: #{$screen__lg + 1px}) {
+            justify-content: flex-start;
+        }
+
+        &::after {
+            content: "";
+            display: inline-block;
+            margin-left: 1rem;
+            width: 54px;
+            height: 30px;
+            background-image: url(/static/images/icons/toggle.svg);
```

Comment:
A future enhancement might be to create the toggle switch in pure css, this would allow us to have a subtle animation of the dot sliding across, but it feels like we could look at this when we do the JS enhancements? (see https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/static_src/sass/components/_toggle-switch.scss / https://rca-development.herokuapp.com/pattern-library/pattern/patterns/atoms/toggle-switch/toggle-switch.html (tbx/tbx), however note that this example is a bit of a frankenstein since it was updated with two "labels"…)

Reply:
The reason I went with the svg was because I was adding it as an `after` element on the label that needs its style to be updated when the sibling element (the checkbox) is clicked or focused, and the style looked too complex to do with one element rather than two. I'm not sure off the top of my head how I'd resolve that but I'll look at your example.

Reply:
Ah Ok I can see how it all works in that RCA example now - the label wraps the input and you have a separate element for the switch.